docs/cmdline-opts/write-out.d

```diff
@@ -39,6 +39,9 @@ option. (Added in 7.26.0)
 The initial path curl ended up in when logging on to the remote FTP
 server. (Added in 7.15.4)
 .TP
+.B header_count
```

Comment:
We use `num_[things]` for connects and redirect. Maybe `num_headers` git better with that scheme?